support-frontend/assets/helpers/render.js

```diff
@@ -60,6 +60,12 @@ const renderPage = (content: Object, id: string, callBack?: () => void) => {
       } else {
         ReactDOM.render(content, element, callBack);
       }
+      setTimeout(() => {
+        if (window.location.hash) {
+          const hashElement = document.getElementById(window.location.hash.substr(1));
+          if (hashElement) { hashElement.scrollIntoView(); }
+        }
```

Comment:
I'd suggest putting this inside a useEffect hook in the Page component (assets/components/page), which I think is the base used for all our pages. That way it should be guaranteed to run after render on any page.